README.md

```diff
@@ -53,20 +53,33 @@ and where.
 ### Manual Installation
 
 1) Pull down the code from https://github.com/SomeOddCodeGuy/OfflineWikipediaTextApi
+   `git clone https://github.com/SomeOddCodeGuy/OfflineWikipediaTextApi`
 2) Open command prompt and navigate to the folder containing the code
+   `cd OfflineWikipediaTextApi`
 3) Optional: create a python virtual environment.
    1) Windows: `python -m venv venv`
    2) MacOS/Linux: `python3 -m venv venv`
 4) Optional: activate python virtual environment.
    1) Windows: `venv\Scripts\activate`
    2) MacOS/Linux: `venv/bin/activate`
+   3) Fish shell: `venv/bin/activate.fish`
 5) Pip install the requirements from requirements.txt
    1) Windows: `python -m pip install -r requirements.txt`
-   2) Linux/MacOS: `python3 -m pip install -r requirements.txt`
+   2) Linux/MacOS: `python -m pip install -r requirements.txt`
 6) Pull down the two needed datasets into the following folders within the project folder:
-   1) `wiki-dataset` folder: https://huggingface.co/datasets/NeuML/wikipedia-20240101
-   2) `txtai-wikipedia` folder: https://huggingface.co/NeuML/txtai-wikipedia
-   3) See project structure below to make sure you did it right
+   1) `wiki-dataset` folder: https://huggingface.co/datasets/NeuML/wikipedia-20240101 
+        You would need git-lfs installed to clone it
+        Windows: https://git-lfs.com/
+        Mac: https://git-lfs.com/ or `brew install git-lfs`
+        Linux Ubuntu/Debian: `sudo apt install git-lfs`
+        Then run:
+        `git lfs install`
+        `git clone https://huggingface.co/datasets/NeuML/wikipedia-20240101`
+        The dataset requieres to be called `wiki-dataset` so rename it:
+        `mv wikipedia-20240101 wiki-dataset`      
+   3) `txtai-wikipedia` folder: https://huggingface.co/NeuML/txtai-wikipedia
+        `git clone https://huggingface.co/NeuML/txtai-wikipedia`
+   5) See project structure below to make sure you did it right
```

Comment:
The instructions all look great but just your numbering got messed up a little bit here.